### ISSUE TITLE
Fullscreen borderless does not work with multiple monitor setups

### ISSUE THREAD

**Issue by ziplock9000** [Beamdog/nwn-issues#643]
### OS

Windows

### Game version

v88.8193.36-12 [d77dd024]

### Describe the crash

While running the game in windowed mode I put the screen into borderless windowed mode.
The game immediately becomes visually static, but clicking actions can be heard in the UI the process needs to be killed via task manager
Starting the game up again, (which will now be in borderless window mode) makes the game hang on start with a white screen. This is 100% reproducible.
The same thing happens when trying to put the game into exclusive mode as well, but I'm not concerned with that.
I can only get the game to work in windowed mode.

Some notes:

This happens in 1080p and 4K modes, 100% and 150% Windows UI scaling, 1x and 2.5x NWN:EE UI Scaling
Completely deleting the settings.xml and manually setting everything back does not help either
Disabling all mods with -noworkshop in Steam does not help.
AMD 5700XT GPU, Catalyst 24.6.1, W11

Finding the cause
I have 3 displays with different resolutions. Even setting them all to 1080p does not help.
But to experiment, I disabled all of them apart from my main one and borderless worked.

**Comment by Finaldeath** [comment]
I can't reproduce but I'm on NVIDIA.

Out of interest how did you change it from windowed to fullscreen/borderless? I tried with alt+enter and it worked fine, as did the menu options.

When it hangs with a white screen can you post the log file?

**Comment by ziplock9000** [comment]
I change it within the Options, Game Options, Graphics.

The UI then becomes static, but the game still runs (I hear clicks). So I have to kill the process.

Alt-Enter is the same.. The whole screen is static but the game sounds and clicking etc can be heard.

When loading the game up again it just hangs on start, black or white screen. It's not just the UI, the whole process hangs.

I'm wondering if it's getting confused about which is the primary monitor.

Logs are mostly empty.

[nwclientError1.txt](https://github.com/user-attachments/files/16145284/nwclientError1.txt)
[nwclientLog1.txt](https://github.com/user-attachments/files/16145285/nwclientLog1.txt)

**Comment by Finaldeath** [comment]
It could be a driver issue that's probably the most common issue. A clean install of GPU drivers may help. AMD has a tool to do this; https://www.amd.com/en/resources/support-articles/faqs/GPU-601.html

If you can change the setting and still type, pressing ``` then typing `pleasecrash` might generate a crash dump.

**Comment by ziplock9000** [comment]
Driver reset didn't work.

I also checked with another OpenGL game in borderless and that worked fine.

I performed `pleasecrash` after changing to borderless. There seems to be no crash dump file as mentioned in the wiki 'Instantly crashes the game. Doesn't seem to generate a crash dump in the documents folder.'

I've included the log files but changed keys

Is there a separate crash file you know of?

[nwclientError1.txt](https://github.com/user-attachments/files/16146149/nwclientError1.txt)
[nwclientlog1.txt](https://github.com/user-attachments/files/16146150/nwclientlog1.txt)

**Comment by Finaldeath** [comment]
Yeah it might need updating to the latest preview for better logs although I guess it will just say it's trying to change it to fullscreen but failing. You can find that on the NWN Developer Discord linked to here: https://discord.gg/CukSHZq

I am not sure there are better ways to debug the AMD drivers over this though.
